R/estimateDiversity.R

```diff
@@ -102,27 +102,30 @@
 #' se <- GlobalPatterns
 #'
 #' # All index names as known by the function
-#' index_code <- c("shannon","gini_simpson","inverse_simpson", "coverage", "fisher")
+#' index <- c("shannon","gini_simpson","inverse_simpson", "coverage", "fisher")
+#'
+#' # Corresponding polished names
+#' name <- c("Shannon","GiniSimpson","InverseSimpson", "Coverage", "Fisher")
```

Comment:
Do you want to use in the later call to `estimateDiversity`?

Reply:
The idea was to give a clear example on how the user can provide both index and name at one go, side by side. They are both used later in the example.